smart_card_connector_app/src/window-about-showing.js

```diff
@@ -46,7 +46,16 @@ const openAboutElement =
 function openAboutClickListener(e) {
   e.preventDefault();
 
-  chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP) {
+    chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  } else if (GSC.Packaging.MODE == GSC.Packaging.Mode.EXTENSION) {
+    chrome.windows.create({
+      url: ABOUT_WINDOW_URL,
+      type: 'popup',
+      width: ABOUT_WINDOW_OPTIONS['innerBounds']['width'],
```

Comment:
This looks weird - let's just create constants ABOUT_WINDOW_WIDTH/ABOUT_WINDOW_HEIGHT instead of ABOUT_WINDOW_OPTIONS.